<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCU_IAP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>串口号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>54</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_baudrate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string>19200</string>
    </property>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_open">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>1001</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_select_file">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>100</y>
      <width>75</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>固件选择</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_bin_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>520</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>550</width>
      <height>40</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start_updata">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>140</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>开始升级</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_mcu">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>140</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设备重启</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect_mcu">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>连接芯片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset_mcu_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>140</y>
      <width>75</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>旧指令</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_set_unixtime">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>75</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>校准时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最大脉冲电压2.0V</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>511</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>设备校准</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>实测脉冲电压：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_pluse_voltage">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>系数：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_coeff">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>40</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_calculate_coeff">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>75</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>计算系数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_set_calib">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>写入校准参数</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_read_calib">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>读取校准参数</string>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_4</zorder>
     <zorder>lineEdit_pluse_voltage</zorder>
     <zorder>label_5</zorder>
     <zorder>lineEdit_coeff</zorder>
     <zorder>pushButton_calculate_coeff</zorder>
     <zorder>pushButton_set_calib</zorder>
     <zorder>pushButton_read_calib</zorder>
    </widget>
   </widget>
   <zorder>frame</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox_port</zorder>
   <zorder>comboBox_baudrate</zorder>
   <zorder>pushButton_open</zorder>
   <zorder>textEdit</zorder>
   <zorder>pushButton_select_file</zorder>
   <zorder>lineEdit_bin_path</zorder>
   <zorder>progressBar</zorder>
   <zorder>pushButton_start_updata</zorder>
   <zorder>pushButton_reset_mcu</zorder>
   <zorder>pushButton_refresh</zorder>
   <zorder>pushButton_connect_mcu</zorder>
   <zorder>pushButton_reset_mcu_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
